src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -175,8 +175,9 @@ def _convert_assets(self, network: Network) -> List[AssetAbstract]:
             if isinstance(esdl_asset.esdl_asset, esdl_junction):
                 continue
             py_assets_list.append(EsdlAssetMapper.to_entity(esdl_asset))
-            py_assets_list[-1].add_physical_data(esdl_asset=esdl_asset)
             network.add_existing_asset(py_assets_list[-1].solver_asset)
+
+        return py_assets_list
```

Comment:
CHeck whether there is one return statement; in my review I see two return statements for this method.